components/use-item-submit.js

```diff
@@ -101,6 +101,7 @@ export function useRetryCreateItem ({ id }) {
   const [retryPaidAction] = usePaidMutation(
     RETRY_PAID_ACTION,
     {
+      alwaysShowQROnFailure: true,
```

Comment:
when the retry button is pressed, we always fallback to a qr code as last resort

Reply:
The reason this isn't the default is that payments don't fail immediately. What's the behavior here when the payment fails 2 minutes later?

Does this prevent a QR from popping up on another, unrelated page 2 minutes later?

Reply:
I thought, that since this is happening after pressing manually on the retry button, the user was going to wait for it. But you are right it is bad ux unless there is a modal or something that tells the user to wait. 
I've restored the previous behavior, better not add more unrelated stuff to this pr